api/v1alpha1/sparkcluster_webhook.go

```diff
@@ -252,11 +252,7 @@ func (sc *SparkCluster) validateFrameworkConfigs() field.ErrorList {
 func (sc *SparkCluster) validateKeyTabConfigs() field.ErrorList {
 	var errs field.ErrorList
 
-	if err := sc.validateKeyTabConfig(sc.Spec.Master.KeyTabConfig, "master"); err != nil {
-		errs = append(errs, err...)
-	}
-
-	if err := sc.validateKeyTabConfig(sc.Spec.Worker.KeyTabConfig, "worker"); err != nil {
+	if err := sc.validateKeyTabConfig(sc.Spec.KerberosKeytab); err != nil {
```

Comment:
Is it possible to use my _shared validation func_ here?

https://github.com/dominodatalab/distributed-compute-operator/blob/3aeef3d7a76c3b4f3e21cd36b7ca38ad62c1dc86/api/v1alpha1/validations.go#L153